packages/gestalt/src/RadioButton.test.js

```diff
@@ -5,28 +5,35 @@ import RadioButton from './RadioButton.js';
 
 test('RadioButton', () => {
   const tree = create(
-    <RadioButton id="id" onChange={() => {}} value="" />
+    <RadioButton id="id" label="Name" onChange={() => {}} value="" />
```

Comment:
@AlbertCarreras let's also add a test for a `RadioButton` without a `label`

Reply:
DONE